qemu/tests/virtio_serial_file_transfer.py

```diff
@@ -148,8 +148,8 @@ def check_output(sender, output):
 
 
 @error_context.context_aware
-def transfer_data(params, vm, host_file_name=None, guest_file_name=None,
-                  sender='both'):
+def transfer_data(params, vm, sender='both', host_file_name=None,
+                  guest_file_name=None):
```

Comment:
You don't need change the order, `transfer_data(params, vm, sender=sender)` will be okay.